qtaf-data/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>de.qytera</groupId>
         <artifactId>qtaf</artifactId>
-        <version>0.0.9</version>
+        <version>LOCAL_2023_04_27_001</version>
```

Comment:
Please don't push `pom` files containing development versions.

Reply:
Set version to 0.1.0